front/tests/008-allowances.spec.ts

```diff
@@ -0,0 +1,108 @@
+
+    await scenarioPage.setTrainScheduleName('Train Schedule 1');
+
+    // ***************** Select Rolling Stock *****************
+    const playwrightRollingstockModalPage = new PlaywrightRollingstockModalPage(
+      playwrightHomePage.page
+    );
+    await playwrightRollingstockModalPage.openRollingstockModal();
+
+    const rollingstockCard = playwrightRollingstockModalPage.getRollingstockCardByTestID(
+      VARIABLES.rollingstockTestID
+    );
+    await rollingstockCard.click();
+
+    await rollingstockCard.locator('button').click();
+
+    // ***************** Select Origin/Destination *****************
+    const playwrightMap = new PlaywrightMap(playwrightHomePage.page);
+    await scenarioPage.openTabByText('Itinéraire');
+    await playwrightMap.page.waitForTimeout(2000);
+    const itinerary = scenarioPage.getItineraryModule;
+    await expect(itinerary).toBeVisible();
+    await expect(scenarioPage.getMapModule).toBeVisible();
+
+    // Search and select origin
+    await playwrightMap.selectOrigin(VARIABLES.originSearch);
+
+    // Search and select destination
+    await playwrightMap.selectDestination(VARIABLES.destinationSearch);
+
+    await scenarioPage.checkPathfindingDistance(VARIABLES.pathfindingDistance);
+
+    // ***************** Select add train *****************
+
+    await scenarioPage.addTrainSchedule();
+    await scenarioPage.returnSimulationResult();
+    await playwrightMap.page.waitForTimeout(2000);
+
+    await expect(scenarioPage.getTimetableList).toBeVisible();
+
+    await scenarioPage.clickBtnByName('Afficher/masquer le détail des trains');
+
+    await scenarioPage.getBtnByName(/Train Schedule 1/).hover();
+
+    await scenarioPage.clickBtnByName('Modifier');
+
+    // ***************** Test apply allowances *****************
+
+    await scenarioPage.openAllowancesModule();
+    await expect(scenarioPage.getAllowancesModule).toBeVisible();
+
+    // Add and check standard allowance
+    await scenarioPage.setStandardAllowance();
+    await scenarioPage.clickBtnByName('Modifier le train');
+
+    expect(await scenarioPage.isAllowanceWorking()).toEqual(true);
+
+    await scenarioPage.getBtnByName(/Train Schedule 1/).hover();
+    await scenarioPage.clickBtnByName('Modifier');
+    await scenarioPage.openAllowancesModule();
+    await expect(scenarioPage.getAllowancesModule).toBeVisible();
+
+    // Add and check engineering allowance
+    await scenarioPage.setEngineeringAllowance();
+    await scenarioPage.clickSuccessBtn();
+    expect(scenarioPage.checkAllowanceEngineeringBtn());
+    await scenarioPage.clickBtnByName('Modifier le train');
+    await scenarioPage.page.waitForSelector('.scenario-details-name');
+
+    expect(await scenarioPage.isAllowanceWorking()).toEqual(true);
+
+    // Delete train
+    await scenarioPage.getBtnByName(/Train Schedule 1/).hover();
+    await scenarioPage.page.getByRole('button', { name: 'Supprimer' }).click();
+  });
```

Comment:
This should be moved in a afterEach/afterAll hook I think (this way, even if the test fails, the trainSchedule is removed).
Also it would be nice to put the main setup in a beforeEach hook, and to keep in the test only the test (start the real test from 
```
    await expect(scenarioPage.getTimetableList).toBeVisible();
    await scenarioPage.clickBtnByName('Afficher/masquer le détail des trains');
    await scenarioPage.getBtnByName(/Train Schedule 1/).hover();
    await scenarioPage.clickBtnByName('Modifier');
```
).

```suggestion
    // Delete train
    await scenarioPage.getBtnByName(/Train Schedule 1/).hover();
    await scenarioPage.page.getByRole('button', { name: 'Supprimer' }).click();
  });
```